goverge/main.py

```diff
@@ -58,18 +58,27 @@ def get_project_package(project_root, project_import):
     return project_import.replace("'", "")
 
 
-def get_test_packages(project_root):
+def get_test_packages(project_root, ignore):
     """
     Get the list of packages in the project
     :type project_root: string
     :param project_root: The location of the project root
     :rtype: List
+    :param ignore: List of directories to ignore
+    :rtype: List
     :return: The list of packages to run coverage on
     """
-    return [
-        x[0] for x in os.walk(project_root)
-        if not any(word in x[0] for word in ["/.", "Godeps", "vendor"])
-        ]
+    ignores = ["/.", "Godeps", "vendor"]
+    if ignore is not None:
+        ignores.extend(ignore)
+    directories = [project_root]
+    for root, subdirs, files in os.walk(project_root):
+        for name in subdirs:
```

Comment:
I'd make this `for subdir in subdirs` as that's what you are doing.

`name` is actually a subdir, not a name, but that makes line 79 confusing when reading with the docstring